packages/light-client/src/index.ts

```diff
@@ -160,22 +160,10 @@ export class Lightclient {
   }
 
   /**
-   * @returns a `Promise` that will resolve once `LightclientEvent.statusChange` with `RunStatusCode.started` value is emitted
+   * @returns a `Promise` that will resolve once `runStatus` equals `RunStatusCode.started`
    */
   start(): Promise<void> {
-    const startPromise = new Promise<void>((resolve) => {
-      const lightclientStarted = (status: RunStatusCode): void => {
-        if (status === RunStatusCode.started) {
-          this.emitter.off(LightclientEvent.statusChange, lightclientStarted);
-          resolve();
-        }
-      };
-      this.emitter.on(LightclientEvent.statusChange, lightclientStarted);
-    });
-
-    void this.runLoop();
-
-    return startPromise;
+    return this.runLoop();
```

Comment:
The problem I see in this approach is that normally user will use `await client.start()`, that will block the event loop forever. 

The syntax `void this.runLoop()` is a way to put the run loop in the background so the `await client.start()` does not block.

Reply:
Makes sense, so I guess the previous approach made more sense.

Reply:
Can't we just check if light client is already started and return right away when `start()` is called?